bst_plugin/process_nst_compute_glm.m

```diff
@@ -178,32 +181,31 @@
     Y = DataMat.F(nirs_ichans,:)';
 
     fitting_choice=cell2mat(sProcess.options.fitting.Value(1));
-    if( fitting_choice == 1 ) % Use OLS : : \( B= ( X^{T}X)^{-1} X^{T} Y \)  
-         B=ols_fit( Y, X );
+    if( fitting_choice == 1 ) % Use OLS : : \( B= ( X^{T}X)^{-1} X^{T} Y \)
+         method_name='OLS_fit';
+         [B,covB,dfe]=ols_fit( Y, X );
+    elseif( fitting_choice == 2 ) % Use AR-IRLS 
+        method_name='AR-IRLS_fit';
+        [B,covB,dfe]=ar_irls_fit( Y, X, round(4/(DataMat.Time(2)-DataMat.Time(1))) ); 
```

Comment:
You can add some check about the availability of nirs-toolbox here and complain if not.
See [example here]( https://github.com/Nirstorm/nirstorm/blob/1e7d4c45bac35411e5de7c1d497bfe7fbebffac3/bst_plugin/process_nst_OM_from_head.m#L139)